docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -405,6 +414,25 @@ this category.
 | v1 | Secret | true |
 | v1 | Service | true |
 
+Note that some `ConfigMap` and `Secret` objets are treated
+differently, as explained in the next section.
+
+#### System infrastructure objects
+
+Even in a kcp workspace, some certain objects --- called here "system
+infrastructure objects" --- are created as a consequence of certain
+other objects or things.  The system infrastructure objects are
+tolerated in the center and do not propagate toward the edge.  Here is
+an initial list of system infrastructure objects:
+
+- `Secret` objects whose _type_ is
+  `kubernetes.io/service-account-token` (these are automatically
+  created to support a `ServiceAccount` in some circumstances) or
+  `bootstrap.kubernetes.io/token`;
+- `ConfigMap` objects named `kube-root-ca.crt`;
+- `ServiceAcount` objects named `default` (these are automatically
+  created created as a consequence of a namespace being created).
```

Comment:
Automatically created twice :)

Reply:
Yes. Two different kinds of objects here are automatically created.

Reply:
Fixed in latest push